sp_mocks/wgp/frontend/package.json

```diff
@@ -9,7 +9,7 @@
   },
   "dependencies": {
     "autoprefixer": "^10.4.16",
-    "layer8_interceptor": "^0.0.17",
+    "layer8_interceptor": "^0.0.22",
```

Comment:
In the imsharer, you don't explicitly put the interceptor in the package.json.
Why here?

Reply:
You're right, I added that, this needs to be there because with that, npm install works correctly and we don't have to install middleware and interceptor with separate commands, changed that, now more easy to install.